extension/manifest.json

```diff
@@ -25,6 +25,21 @@
         "48": "images/GearHeart_Mixed_47x47.png",
         "128": "images/GearHeart_Mixed_127x127.png"
     },
+    "commands": {
+        "toggle-feature": {
+            "suggested_key": {
+                "default": "Ctrl+Shift+F",
+                "mac": "MacCtrl+Shift+F"
+            },
+            "description": "Send a 'toggle-feature' event to the extension"
+        },
+        "_execute_browser_action": {
+            "suggested_key": {
+                "default": "Ctrl+Shift+Y",
+                "mac": "MacCtrl+Shift+Y"
```

Comment:
it's probably fine to leave it at the default. Mac users are used to Ctrl being Command on macOS.

Reply:
I think I'm not able to understand. Do you mean I should use "Command" instead of "MacCtrl" as "Ctrl" gets converted to "Command" automatically on mac systems?

Reply:
The suggestion for the keyboard shortcut was to use a "u" have you tried that instead of the "y"? see: https://issues.gpii.net/browse/GPII-4241

Reply:
> I think I'm not able to understand. Do you mean I should use "Command" instead of "MacCtrl" as "Ctrl" gets converted to "Command" automatically on mac systems?

From the [chrome.commands](https://developer.chrome.com/apps/commands) docs, "Please note that on Mac 'Ctrl' is automatically converted to 'Command'." You can just leave the default. I don't think we need it to be the CTRL key on macOS. Most application that use ctrl+ on windows use command+ on macOS.

Reply:
I can be changed to "U", it'll work for "Ctrl+Shift+U" as well. Shall I change it?

Reply:
yes, please change to "u"